introduction.md

```diff
@@ -1 +1,6 @@
-Hello i'm Mateus, my favorite Code School path is Ruby.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?